app/controllers/projects_controller.rb

```diff
@@ -4,11 +4,15 @@ class ProjectsController < ApplicationController
     before_action :authenticate_user!
 
     def index
-        if (current_user.role=="manager")
-            
-            @projects=current_user.created_projects
+        if (current_user.role == "manager")
+            @projects = current_user.created_projects.paginate(page: params[:page], per_page: 3)
```

Comment:
@Usmanksri add per_page in the project model. Look will paginate documentation
https://github.com/mislav/will_paginate